website/docs/d/karbon_cluster.html.markdown

```diff
@@ -0,0 +1,102 @@
+---
+layout: "nutanix"
+page_title: "NUTANIX: nutanix_karbon_cluster"
+sidebar_current: "docs-nutanix-datasource-karbon-cluster"
+description: |-
+ Describes a Karbon Cluster
+---
+
+# nutanix_karbon_cluster
+
+Describes Karbon Clusters
+
+## Example Usage
+
+```hcl
+data "nutanix_karbon_cluster" "cluster" {
+   cluster_id = "<YOUR-CLUSTER-ID>"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `karbon_cluster_id`: Represents karbon cluster uuid
+* `karbon_cluster_name`: Represents the name of karbon cluster
+
+## Attribute Reference
+
+The following arguments are supported:
+
+* `name`: - The name for the k8s cluster.
+* `wait_timeout_minutes`
+* `version`: - K8s version of the cluster.
+* `storage_class_config`
```

Comment:
Storage class configuration attribute for defining the persistent volume attributes.